packages/algolia/test/client.test.ts

```diff
@@ -15,7 +17,33 @@ describe('Algolia Search Client', () => {
   const algoliaSearchIndex = getAlgoliaSearchIndexMock();
   const algoliaSearchClient = new AlgoliaSearchClient(algoliaSearchIndex);
 
-  test('Should save the Research Output', async () => {
+  it('Should do batch on entities', async () => {
+    await algoliaSearchClient.batch([
```

Comment:
since it's a batch request i would test it with an actual batch (say 2 or 3 elements), not just one element in that array